index.html

```diff
@@ -1,187 +1,284 @@
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-                <div class="card card4">
-                    <div class="image-card-holder"></div>
-                    <div class="text-work-title">
-                        <p>Multi-Post Stories Gain+Glory</p>
-                    </div>
-                    <div class="card-tags">
-                        <ul>
-                            <li class="tag-element">Ruby on rails</li>
-                            <li class="tag-element">css</li>
-                            <li class="tag-element">JavScript</li>
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-                <div class="card card5">
-                    <div class="image-card-holder"></div>
-                    <div class="text-work-title">
-                        <p>Multi-Post Stories Gain+Glory</p>
-                    </div>
-                    <div class="card-tags">
-                        <ul>
-                            <li class="tag-element">Ruby on rails</li>
-                            <li class="tag-element">css</li>
-                            <li class="tag-element">JavScript</li>
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-                <div class="card card6">
-                    <div class="image-card-holder"></div>
-                    <div class="text-work-title">
-                        <p>Multi-Post Stories Gain+Glory</p>
-                    </div>
-                    <div class="card-tags">
-                        <ul>
-                            <li class="tag-element">Ruby on rails</li>
-                            <li class="tag-element">css</li>
-                            <li class="tag-element">JavScript</li>
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-            </div>   
-        </section>
+          </div>
+        </div>
+      </section>
+
+      <section class="form-contact">
+        <h3 class="text-contact">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+        <form
+          id="template-form"
+          action="https://formspree.io/f/template-form"
+          method="post"
```

Comment:
- [ ] Your project is looking great! However, the link for the `action` attribute from the `form` is still not working, as the last reviewer said, this link should look something like: `https://formspree.io/f/{form_id}`.

If you have already updated it please don't forget to push your commits before submitting a new code review.

Reply:
@smunozmo yeah actually my mistake there I lost my last submit what can I do? everything works fine